src/allotropy/parsers/lines_reader.py

```diff
@@ -132,19 +132,15 @@ def pop_csv_block_as_lines(self, empty_pat: str = EMPTY_STR_PATTERN) -> list[str
     def pop_csv_block_as_df(
         self,
         empty_pat: str = EMPTY_STR_PATTERN,
-        *,
-        header: Optional[Union[int, Literal["infer"]]] = None,
-        sep: Optional[str] = ",",
-        as_str: bool = False,
```

Comment:
confirming that this functionality is no longer used/needed?

Reply:
yes, this values are no longer needed